src/components/investigate-dialog.tsx

```diff
@@ -13,32 +13,32 @@ export class InvestigateDialogComponent extends BaseComponent<IProps, IState> {
 
   public render() {
     return (
-      <div className="investigate-dialog">
+      <div className="investigate-dialog" data-test="investigate-dialogue">
```

Comment:
One more place where we should use "dialog" (there are a few more, but I'll stop littering this with redundant comments).